bundle/docsgen/main.go

```diff
@@ -137,6 +137,14 @@ func assignAnnotation(s *jsonschema.Schema, a annotation.Descriptor) {
 		s.Deprecated = true
 		s.DeprecationMessage = a.DeprecationMessage
 	}
+	if a.ForceNotDeprecated {
+		s.Deprecated = false
+		s.DeprecationMessage = ""
+	}
+	if a.Preview == "PRIVATE" {
+		s.DoNotSuggest = true
+		s.Preview = a.Preview
+	}
```

Comment:
Why does this logic need to be duplicated between this instance and in `bundle/internal/schema/annotations.go`?

@ilyakuz-db

Reply:
I think we can exclude this change, but overall, we should be following the same pattern as for Sphinx docs in docsgen

Reply:
Sounds good.